aliases/available/uuid.aliases.bash

```diff
@@ -0,0 +1,4 @@
+# uuidgen
```

Comment:
Can you please add the `cite` and `about-alias` headers? See here for an example:

https://github.com/Bash-it/bash-it/blob/9e92ef342f9673559e78220b14cd535aa631f4c7/aliases/available/ag.aliases.bash#L1-L2

Reply:
Didn't see this -- now done.

Reply:
Sorry, can't see your change - did you push it to this branch to update the PR?

Reply:
Thought I had.  Made one more change, since most alias files seem to be named based on the actual command, which is `uuidgen`:  moved `uuid.aliases.bash` to `uuidgen.aliases.bash`.

Reply:
Should be done now @nwinkler.

Reply:
Thanks - good point on the file name!